### ISSUE TITLE
Large loss with minor decrease

### ISSUE THREAD

**Issue by DeepRunner** [GengDavid/pytorch-cpn#14]
Hi.
I train 256.192.model from scratch on COCO keypoint 2017. What confused me is that the average loss starts with large numbers(e.g.400) and with minor decrease(decrease 2 every 100 iterations with batch size 32). Is this a normal?
Thanks!

**Comment by GengDavid** [comment]
Maybe you can refer to [this log](https://user-images.githubusercontent.com/16283388/48302061-37acd800-e532-11e8-897c-6d5ceeeac9de.png).
